systemtests/generate_testdata_on_network.py

```diff
@@ -608,6 +578,11 @@ def do_create_contract_deployments(self):
             contract_address = address_computer.compute_contract_address(sender, transaction.nonce)
             self.memento.add_contract("dummy", contract_address.to_bech32())
 
+        for transaction in transactions_forwarder:
+            sender = Address.from_bech32(transaction.sender)
```

Comment:
perhaps use `Address.new_from_bech32()` in the future.

Reply:
Good point!